src/main/java/net/snowflake/client/jdbc/SnowflakeSQLException.java

```diff
@@ -48,41 +49,49 @@ public SnowflakeSQLException(String queryId,
 
     // log user error from GS at fine level
     logger.debug("Snowflake exception: {}, sqlState:{}, vendorCode:{}, queryId:{}",
-                 reason, sqlState, vendorCode, queryId);
+            reason, sqlState, vendorCode, queryId);
+
+  }
 
+  public SnowflakeSQLException(String reason, String SQLState)
```

Comment:
Inheritance goes SQLException-> SnowflakeSQLException -> SnowflakeSQLLoggedException.
Now, exceptions of type SQLException using a constructor previously missing from SnowflakeSQLException can be refactored as SnowflakeSQLLoggedExceptions.